src/algebra/chinese-remainder-theorem.md

```diff
@@ -15,7 +15,7 @@ Let $m = m_1 \cdot m_2 \cdots m_k$, where $m_i$ are pairwise coprime. In additio
 $$\begin{align}
     a &\equiv a_1 \pmod{m_1} \\
     a &\equiv a_2 \pmod{m_2} \\
-      &\ldots \\
+      & \vdots \\
```

Comment:
Sadly, this looks a bit off. The dots are not aligned properly.
![image](https://user-images.githubusercontent.com/7143573/214953141-6a5658cc-a191-4a8b-9fcc-7e89f671bc9c.png)

A fix would be to use an array instead of align:
```tex
$$\begin{array}{rcl}
    a & \equiv & a_1 \pmod{m_1} \\
    a & \equiv & a_2 \pmod{m_2} \\
      & \vdots & \\
    a & \equiv & a_k \pmod{m_k}
\end{array}$$
```
![image](https://user-images.githubusercontent.com/7143573/214953258-ceb24942-1b5c-4c5c-926a-75c14d1d659a.png)